src/lib/tls/ocsp.c

```diff
@@ -331,6 +331,11 @@ int tls_ocsp_check(REQUEST *request, SSL *ssl,
 		break;
 	}
 
+	if (issuer_cert == NULL) {
+		REDEBUG("ocsp: couldn't get issuer certificate");
+		goto skipped;
```

Comment:
this will print "tls - oscsp - oscsp:", and it's not really an error in that it doesn't signal a fatal condition, so it's more RWDEBUG.

Reply:
Switched to RWDEBUG, without prefix.